tests/playwright/conftest.py

```
@@ -653,3 +653,76 @@ def pywebview_mock(page: Page) -> PywebviewMockHandle:  # type: ignore[name-defi
+    """Ensure the Playwright session event loop is set as the running loop for this test.
+
+    This counters the 'clear_leaked_running_loop' autouse fixture in the root conftest,
+    which clears the running loop to prevent conflicts in other async tests.
+    """
+    import asyncio.events
+
+    if _PLAYWRIGHT_LOOP is not None:
+        try:
+            asyncio.events._set_running_loop(_PLAYWRIGHT_LOOP)
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)